smallrye-reactive-messaging-kafka/src/test/java/io/smallrye/reactive/messaging/kafka/MetadataPropagationTest.java

```diff
@@ -120,6 +140,27 @@ private KafkaMapBasedConfig getKafkaSinkConfigForMyAppProcessingData(String topi
         return builder.build();
     }
 
+    private KafkaMapBasedConfig getKafkaSinkConfigForAppProcessingDataForwardKey(String topicOut, String topicIn) {
+        KafkaMapBasedConfig.Builder builder = KafkaMapBasedConfig.builder();
+        builder.put("mp.messaging.outgoing.kafka.value.serializer", StringSerializer.class.getName());
```

Comment:
`IntegerSerializer`?

Reply:
Well it doesn't make any difference for the test :)